woocommerce/woocommerce
# update chile regions

### All Submissions:






### Changes proposed in this Pull Request:
add chile regions based on https://en.wikipedia.org/wiki/ISO_3166-2:CL



Closes #30375

### How to test the changes in this Pull Request:

1. setup a new store
2. check regions added for chile f.e. on checkout

### Other information:




### Changelog entry

> Tweak - Improve internationalization and add regions for Chile.

### FOR PR REVIEWER ONLY:
